opennms-services/src/main/java/org/opennms/netmgt/notifd/MattermostNotificationStrategy.java

```diff
@@ -80,8 +80,11 @@ protected String formatWebhookErrorResponse(int statusCode, String contents) {
 
     @Override
 	protected String decorateMessageSubject(String subject) {
-    	StringBuilder bldr = new StringBuilder("#### ");
-    	bldr.append(subject).append("\n");
+    	if ("".equals(subject)) {
+    		return "";
+    	}
+    	StringBuilder bldr = new StringBuilder("*");
```

Comment:
If you're going for bold, Mattermost wants two asterisks to Slack's one.
https://api.slack.com/docs/message-formatting#message_formatting
https://docs.mattermost.com/help/messaging/formatting-text.html#text-style

Reply:
Good catch! Totally missed that one!